### DISCUSSION TITLE
btt tft50 v3.0/btt skr2 rev.b

### DISCUSSION THREAD

**Discussion by 420kid** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2315] [Q&A]
Hello I have the ender 5 plus all factory except I installed the btt skr2 rev.b and btt tft50 v3.0 my problem is that the touch screen has no reaction to touch and on the marlin side I get a blank screen the dial works only on the marlin/touch menu. Ihave updated the screen firmware and it did so just fine I have also updated marlin firmware on main board. I have been working on this for months now and can't seem to figure out where I am going wrong. So I am here for help from the community. If anyone can lead me in the right direction I would be much appreciated. please help

**Comment by kisslorand** [comment]
Try with "reset.txt" on your SDCard.

**Reply by 420kid** [reply]
I guess I'm not understanding that process to reset can you give me more details on how to do that cause screen is seeing that I have inserted the sd card but that bout it. I'm not doing something right. I'm kinda new at the 3d printing thing and computers I'm not the sharpest pencil in the box either. thanks for your quick response

**Comment by kisslorand** [comment]
Make an empty text file called "reset.txt" and copy it to an SDCard. Than insert that SDCard into your TFTs SDCard reader and power on the TFT.

**Comment by oldman4U** [comment]
Front page of this repository. ReadMe. Troubleshooting. Does not help - broken hardware.

**Reply by 420kid** [reply]
Broken hardware ? would you be talking main board? If so how would I get it taken care of ? I bought it late summer early fall and have yet to get either of the 2 pieces to work even using a combination of board and screen I have my original ender 5+ screen and the TFT50 V3.0 and probably 5 or 6 different boards I need details sorry. but new I put on hardware all day at work and this has nothing to do with doors, kitchens or offices that I'm sure of.

**Comment by 420kid** [comment]
Sorry for the slow response been super busy with work. Thanks for the advice I did get the screen to reset I think it didn't do anything special so not 100%. but I am still having trouble getting things to work so couple  ?'s I have been setting up code for the BTT SKR2_0 with the TFT50 V3_0 I have went over every word of the code and can't see where I may have went wrong with that said every time I look up BTT SKR2... I see a lot of references about the BTT SKR2 E3 mini if I remember correctly and I'm sure of the board but again I am such a noob when it comes to this 3D printing world and coding firmware kind of thing even though It can be daunting and make me want to throw some thing I keep my cool and I find myself enjoying it all (the Challenge). so main board clearly states BTT SKR2... and nothing bout E3 mini or any other clue I could be wrong  so is there a chance I am ? next ? What are the chances I could have a bad board ? I see a lot said in that area. O and my main board has the 2 QR stickers on back side so it's got me thinking I have the BTT SKR2_0 Rev. B. So any more leads on this matter ? I really want my printer back been down for a long time now I'm trying everything ya'll suggest. Again thanks for ya'lls time.

**Comment by radek8** [comment]
Send the configuration and wiring photos.
